app/sections/root/sections/tweets/tweets.component.ts

```diff
@@ -6,7 +10,73 @@ import {Component, Input} from 'angular2/core';
   styles: [
     <string> require('./tweets.styl')
   ],
+  directives: [...CORE_DIRECTIVES],
+  pipes: [ToDate]
 })
-export class TweetsComponent {
-  @Input() author:string;
+export class TweetsComponent implements OnInit {
+  @Input()
+  author:string;
+
+  private infFollow: Object= {
+    link: 'https://twitter.com/hansrosling',
+    title: 'Follow Hans',
+    count: 'Followers'
+  };
+
+  private tweets: Array<any> = [];
```

Comment:
It should be array of tweet objects - and the tweet object should be defined ( I assume that is has exact same structure as in tests example ).  
Also, we should stick to one naming convention - `any[]` or `Array<any>`, @rychkog @natali-abyss which one do you prefer ?